### ISSUE TITLE
Infilling possible?

### ISSUE THREAD

**Issue by avilella** [programmablebio/pepprclip#1]
Hiven a target sequence and an incomplete binder sequence with some 'X' characters in certain places, would pepprclip be able to fill in the X characters for amino-acids that make the best binder sequence? Thx.

**Comment by SuhaasBhat** [comment]
Yes, this is possible with some minor code modifications of the generation step. The simplest approach is to change the parameters of the generation step to make a shorter sequence, and insert that shorter sequence where you would put your ‘X’s. You may also consider modifying the generation algorithm to only perturb your selected ‘X’ positions. 

In any case, the CLIP discriminator will perform well for any method of generating peptides.

**Comment by SuhaasBhat** [comment]
By the way, thank you for using our tool! We’d really appreciate it if you could spread the word about our paper!
